### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by ghost**
leave a comment

### CODE REVIEW

File: thomas/index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 </head>
 <body>
+    <a https://bobobson.github.io/fish/Thomas.html> amazing page</a>
```
Comment (domdomegg) [inline thomas/index.html:22]: You need to use a proper `href` attribute, like other links

**Review by domdomegg — CHANGES_REQUESTED** [review]

**Comment by domdomegg** [comment]
Might also be an idea to use different images for your products

**Comment by jcxldn** [comment]
bump @domdomegg 
Inactive pull.

**Comment by domdomegg** [comment]
Waiting on @bobobson

**Comment by domdomegg** [comment]
Inactive pull, closing.

**Pull request closed without merging**